contrib/libvirt/files/virtstoraged

```diff
@@ -0,0 +1,9 @@
+# virtstoraged daemon service
+# clonficts with libvirtd daemon
```

Comment:
clonflicts

Reply:
(ditto for anywhere else, but i'd just drop the libvirtd service *and* these lines)